src/Main.java

```diff
@@ -7,77 +7,119 @@
+        if (number == '0') {
+            drawLine(img, offset, true, false, true, true, true, true, true, false, false, false);
         }
-        if( w2 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 99, 82, 2);
+        if (number == '1') {
+            drawLine(img, offset, false, false, false, false, false, true, true, false, false, false);
         }
-        if( w3 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 179, 82, 2);
+        if (number == '2') {
+            drawLine(img, offset, true, true, true, false, true, true, false, false, false, false);
         }
-        if( h1 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 19, 2, 82);
+        if (number == '3') {
+            drawLine(img, offset, true, true, true, false, false, true, true, false, false, false);
         }
-        if( h2 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(9, 99, 2, 82);
+        if (number == '4') {
+            drawLine(img, offset, false, true, false, true, false, true, true, false, false, false);
         }
-        if( h3 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(89, 19, 2, 82);
+        if (number == '5') {
+            drawLine(img, offset, true, true, true, true, false, false, true, false, false, false);
         }
-        if( h4 ) {
-            g2d.setColor(Color.BLACK);
-            g2d.fillRect(89, 99, 2, 82);
+        if (number == '6') {
+            drawLine(img, offset, true, true, true, true, true, false, true, false, false, false);
+        }
+        if (number == '7') {
+            drawLine(img, offset, true, false, false, false, false, true, true, false, false, false);
+        }
+        if (number == '8') {
+            drawLine(img, offset, true, true, true, true, true, true, true, false, false, false);
+        }
+        if (number == '9') {
+            drawLine(img, offset, true, true, true, true, false, true, true, false, false, false);
         }
     }
 
-    public static void main(String[] args) throws IOException {
-        BufferedImage img = new BufferedImage(100, 200, BufferedImage.TYPE_INT_RGB);
+    static void drawSymbol(BufferedImage img, char number, int offset) throws IOException {
+        if (number == '-') {
+            drawLine(img, offset, false, false, false, false, false, false, false, true, false, false);
+        }
+        if (number == '+') {
+            drawLine(img, offset, false, false, false, false, false, false, false, false, true, false);
+        }
+        if (number == '=') {
+            drawLine(img, offset, false, false, false, false, false, false, false, false, false, true);
+        }
+    }
+
+    static void drawLine(BufferedImage img, int offset, boolean w1, boolean w2, boolean w3, boolean h1, boolean h2, boolean h3, boolean h4, boolean e1, boolean e2, boolean e3) throws IOException {

```

Comment:
For drawing number there is always false, false, false at the end, for drawing symbols it's the other way around. Consider creating separate methods for drawing lines for numbers, and lines for symbols. I would find it a bit more readable. 
like:
```
static void drawNumberLine(BufferedImage img, int offset, boolean v1, boolean v2, boolean v3, boolean h1, boolean h2, boolean h3, boolean h4)
```
and
```
static void drawSymbolLine(BufferedImage img, int offset, boolean e1, boolean e2, boolean e3)
```
What's more drawLine for symbols now sometimes creates two lines (for + and =) thus it's lines not line.

Even better solution would be to create three methods:
`drawPlus`
`drawMinus`
`drawEqual`

and remove three last parameters from `drawLine`

Thanks to that, we'll get rid of three if statements.

Reply:
Symbols have been split into `drawSymbol` to read the typed characters.
Each symbol got its own method:
`drawPlus`
`drawMinus`
`drawEqual`